weblab_cg/tests/test_chaste_cg_develop.py

```diff
@@ -33,8 +33,6 @@ def chaste_models(self):
         return load_chaste_models(model_types=self.model_types(),
                                   ref_path_prefix=['chaste_reference_models', 'develop'], class_name_prefix='Dynamic')
 
-    @pytest.mark.skip(reason="This test is a development tool")
```

Comment:
The marks have gone here. From dd4873f it seems that was unintentional?

Reply:
I forgot the chaste mark thanks! I deliberately removed the other one as manually changing it was driving me nuts, when there are no reference models it'll be skipped anyway.